site/index.html

```diff
@@ -76,6 +76,52 @@ <h2>Try it!</h2>
 
 <br>
 
+<div class="features">
+   <div class=" container ">
+      <h2>Why Haskell?</h2>
+      <div class=" row ">
+         <div class=" span6 col-md-6">
+            <h3>Rapid prototyping</h3>
+            <p>Start small and explore new ideas to find your market fit. Haskell is a compact language with a large ecosystem of open-source libraries and tools
+               to get you started and help you growing.
+               The strong type system makes it easy to write programs that work, both large and small, even when you don't like writing types &mdash; the compiler will figure it out.
```

Comment:
Perhaps we could also mention that you can defer type checking to run-time, letting you test programs that are not completely finished yet.

Reply:
To me this seems like a lower-level details that could fit into another PR for the "Features" section further below on the site, e.g. the "Statically typed" subsection.